src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+
+  const longTerm = 7;
+  const longTermDiscount = 50;
+
+  let res = 0;
+
+  for (let i = 0; i < days; i++) {
+    res += 40;
+  }
```

Comment:
You don't need this loop. It is enough to multiply the number of days by the constant cost per day.
res = days * 40;

Additionally, you may use this variable 'res' at the beginning and after the final 'return'. 

Also, the name 'res' is not very representative. Something like 'totalCost' or 'totalRent' would be better.